### PR TITLE
[Patch] Fix terminate outdated employee requests test

### PR DESCRIPTION
**Pull request by progsmile**
Fix terminate outdated employee requests test
[Issue](https://github.com/edenlabllc/ehealth.api/issues/1977)

### CODE REVIEW

File: apps/ehealth/test/unit/employee_requests/employee_requests_test.exs [hunk 1]
```diff
@@ -3,53 +3,44 @@ defmodule EHealth.Unit.EmployeeRequestsTest do
...
-    |> Repo.update()
-
-    employee_request2
-    |> Ecto.Changeset.change(inserted_at: NaiveDateTime.add(inserted_at, 10, :seconds))
-    |> Repo.update()
+    inserted_at = Timex.shift(NaiveDateTime.utc_now(), days: -10)
 
-    assert 3 = Request |> Repo.all() |> Enum.count()
+    employee_request_expired = insert(:il, :employee_request, employee_id: employee_id, inserted_at: inserted_at)
+    _employee_request_expired2 = insert(:il, :employee_request, inserted_at: inserted_at)
```
Comment (PavelVesnin) [inline apps/ehealth/test/unit/employee_requests/employee_requests_test.exs]: what for this var `_employee_request_expired2`?

File: apps/ehealth/test/unit/employee_requests/employee_requests_test.exs [hunk 2]
```diff
@@ -3,53 +3,44 @@ defmodule EHealth.Unit.EmployeeRequestsTest do
...
-    request_id = employee_request1.id
-    expired_status = Request.status(:expired)
-    assert %{status: ^expired_status} = Repo.get(Request, request_id)
+    assert %{status: @expired_status} = Repo.get(Request, employee_request_expired.id)
 
-    assert [event1, _event2] =
-             Event
-             |> order_by([e], e.inserted_at)
-             |> EventManagerRepo.all()
+    [event1 | _] = events = EventManagerRepo.all(Event)
```
Comment (PavelVesnin) [inline apps/ehealth/test/unit/employee_requests/employee_requests_test.exs]: lets check all events

File: apps/ehealth/test/unit/employee_requests/employee_requests_test.exs (hunk 1 above)
Comment (progsmile) [reply apps/ehealth/test/unit/employee_requests/employee_requests_test.exs]: I'll remove it

File: apps/ehealth/test/unit/employee_requests/employee_requests_test.exs (hunk 2 above)
Comment (progsmile) [reply apps/ehealth/test/unit/employee_requests/employee_requests_test.exs]: We have 2 events: one with our `employee_id`, another with random id
So, what is expected from 2nd event?

File: apps/ehealth/test/unit/employee_requests/employee_requests_test.exs (hunk 1 above)
Comment (progsmile) [reply apps/ehealth/test/unit/employee_requests/employee_requests_test.exs]: I left it for code readability)

File: apps/ehealth/test/unit/employee_requests/employee_requests_test.exs (hunk 2 above)
Comment (PavelVesnin) [reply apps/ehealth/test/unit/employee_requests/employee_requests_test.exs]: you right, ignore it

File: apps/ehealth/test/unit/employee_requests/employee_requests_test.exs [hunk 3]
```diff
@@ -3,53 +3,44 @@ defmodule EHealth.Unit.EmployeeRequestsTest do
...
+    events_expired_entities_ids = events |> Enum.map(&Map.get(&1, :entity_id))
+    events_expired_properties = events |> Enum.map(&Map.get(&1, :properties))
```
Comment (PavelVesnin) [inline apps/ehealth/test/unit/employee_requests/employee_requests_test.exs]: do not use pipeline for single function call
```Enum.map(events, &Map.get(&1, :properties))```

File: apps/ehealth/test/unit/employee_requests/employee_requests_test.exs (hunk 2 above)
Comment (PavelVesnin) [reply apps/ehealth/test/unit/employee_requests/employee_requests_test.exs]: why you remove assertion here?

**Pull request merged**